x-pack/plugins/snapshot_restore/public/application/sections/home/repository_list/repository_table/repository_table.tsx

```diff
@@ -249,7 +249,11 @@ export const RepositoryTable: React.FunctionComponent<Props> = ({
       </EuiButton>,
       <EuiButton
         key="registerRepo"
-        {...reactRouterNavigate(history, linkToAddRepository(name))}
+        {...reactRouterNavigate(
+          history,
+          // @ts-expect-error
```

Comment:
`name` is not defined in the scope. It uses deprecated global var name from `lib.dom.d.ts`. @elastic/es-ui

Reply:
I think you intended this comment to be on line 49 of `hdfs_settings.tsx`, above, correct?

Reply:
If so, could you clarify the error that's expected on this line?

Reply:
> I think you intended this comment to be on line 40 of hdfs_settings.tsx, above, correct?

TS doesn't complain there, does it? 
![2020-11-24_08-56-41](https://user-images.githubusercontent.com/3198181/100064620-10808500-2e33-11eb-9248-7617ae5f82ce.png)

> If so, could you clarify the error that's expected on this line?

As written in the comment: `name` is not defined in the local scope and references a deprecated property `name` from the global scope.
![2020-11-24_08-56-04](https://user-images.githubusercontent.com/3198181/100064778-3ad24280-2e33-11eb-9d68-c2806837a839.png)

Reply:
I've tested this locally and it looks like that name value can safely be removed. Would you mind removing the TS comment and the reference to the global "name" @restrry ? Otherwise I can do it as a follow up chore. Either way, no need to block on this.

Reply:
I would prefer the owners to remove this code, as some nuances may pop up.